internal/cluster/common_test.go

```diff
@@ -300,34 +300,41 @@ var _ = Describe("UpdateMachineCidr", func() {
 			expectedClusterMachineNetworks: []*models.MachineNetwork{{Cidr: "5.6.7.0/24"}},
 			update:                         true,
 		},
-		// TODO MGMT-7678: Support dual-stack. It requires to indicate primary by a new flag field / ordinal numbering
-		// {
-		// 	name:                           "cluster with multiple machine networks with an empty new value",
-		// 	clusterMachineNetworks:         append(common.TestIPv4Networking.MachineNetworks, common.TestIPv6Networking.MachineNetworks...),
-		// 	expectedClusterMachineNetworks: common.TestIPv6Networking.MachineNetworks,
-		// 	update:                         true,
-		// },
-		// {
-		// 	name:                           "cluster with multiple machine networks with existing primary new value",
-		// 	clusterMachineNetworks:         append(common.TestIPv4Networking.MachineNetworks, common.TestIPv6Networking.MachineNetworks...),
-		// 	newMachineCidr:                 string(common.TestIPv4Networking.MachineNetworks[0].Cidr),
-		// 	expectedClusterMachineNetworks: append(common.TestIPv4Networking.MachineNetworks, common.TestIPv6Networking.MachineNetworks...),
-		// 	update:                         false,
-		// },
-		// {
-		// 	name:                           "cluster with multiple machine networks with existing secondary new value",
-		// 	clusterMachineNetworks:         append(common.TestIPv4Networking.MachineNetworks, common.TestIPv6Networking.MachineNetworks...),
-		// 	newMachineCidr:                 string(common.TestIPv6Networking.MachineNetworks[0].Cidr),
-		// 	expectedClusterMachineNetworks: common.TestIPv6Networking.MachineNetworks,
-		// 	update:                         true,
-		// },
-		// {
-		// 	name:                           "cluster with multiple machine networks with different new value",
-		// 	clusterMachineNetworks:         append(common.TestIPv4Networking.MachineNetworks, common.TestIPv6Networking.MachineNetworks...),
-		// 	newMachineCidr:                 "5.6.7.0/24",
-		// 	expectedClusterMachineNetworks: append([]*models.MachineNetwork{{Cidr: "5.6.7.0/24"}}, common.TestIPv6Networking.MachineNetworks...),
-		// 	update:                         true,
-		// },
+		{
```

Comment:
A proposal, but not a must:
Should those tests also assert that `machine_network_cidr_updated_at` has changed in successful cases and has not changed otherwise?

Reply:
We already check it
```
if test.update {
				Expect(clusterFromDb.MachineNetworkCidrUpdatedAt).NotTo(Equal(cluster.MachineNetworkCidrUpdatedAt))
			} else {
				Expect(clusterFromDb.MachineNetworkCidrUpdatedAt).To(Equal(cluster.MachineNetworkCidrUpdatedAt))
			}
```

Reply:
In that case, please disregard.